AmazonChimeSDK/AmazonChimeSDK/realtime/RealtimeControllerFacade.swift

```diff
@@ -55,4 +55,19 @@ import Foundation
     /// - Parameter lifetimeMs: Message lifetime in milisecond, 5 mins max, default 0
     /// - Throws: SendDataMessageError
     func realtimeSendDataMessage(topic: String, data: Any, lifetimeMs: Int32) throws
+
+    /// Enable/disable Voice Focus (ML-based noise suppression)
+    ///
+    /// Note: this API can only be called after audio session was started.
```

Comment:
nit: after audio session *is* started?

In fact we don't prevent calling the API before audio session starts, should we rephrase to something like "Only call this API after audio session is started, calling it before that results in a no-op"

Reply:
Rephrased as suggested~